src/bootstrap-tenant/bootstrapper/items.ts

```diff
@@ -1229,25 +1238,37 @@ export async function setItems({
       if (item._componentsData?.[language]) {
         Object.keys(item._componentsData[language]).forEach(
           (componentId: string) => {
-            const componentContent: ComponentContentInput =
+            const componentContent: any =
```

Comment:
bad bad bad. Keep the types. There are better ways of doing the bit that comes later.